src/components/CreateVmWizard/steps/BasicSettings.js

```diff
@@ -302,6 +313,7 @@ class BasicSettings extends React.Component {
     const { data, clusters, maxNumOfSockets, maxNumOfCores, maxNumOfThreads, operatingSystems } = this.props
     const indicators = {
       name: this.validateVmName(),
+      hostName: !isHostNameValid(data.initHostname || ''),
```

Comment:
it seems that isHostNameValid validation logic is wrong (only alphabetic chars are accepted). Please compare to webadmin for this

Reply:
It works for me, the rules in web UI are different than web Admin so I changed it according to webAdmin [validation](https://github.com/oVirt/ovirt-engine/blob/e48484ccb3c1c8fcaed81023b234295a109ebd8b/frontend/webadmin/modules/uicommonweb/src/main/java/org/ovirt/engine/ui/uicommonweb/validation/HostnameValidation.java#L9).